scout/server/blueprints/variant/templates/variant/clinvar.html

```diff
@@ -135,14 +135,8 @@ <h5>
                           {% set gene_symbols = [] %}
                           {% for gene in pinned.genes %}
                             {% do gene_symbols.append(gene.hgnc_symbol) %}
-                            {% if pinned.genes|length == 1 %}
-                              <strong>{{ pinned.genes[0].hgnc_symbol }}</strong><br>
-                            {% else %}
-                              {% for gene in pinned.genes %}
-                              <strong>{{ gene.hgnc_symbol+";" }}</strong>
-                              {% endfor %}
-                            {% endif %}
                           {% endfor %}
+                            <strong>{{gene_symbols|join(", ")}}</strong>
```

Comment:
Remind me, we did not enable SV submission yet, right? Because this could potentially grow really, really long then. Optionally truncate at a max len and add like an ellipsis with the number of genes to future proof.

Reply:
We did enable it actually. It could grow big but perhaps let's leave it to the users if they want to include them or not? I'm not really sure how they want it at ClinVar